### DISCUSSION TITLE
i need help with gui i started coding os like 2 days ago i readed whole internet and still idk

### DISCUSSION THREAD

**Discussion by NikitaPos** [CosmosOS/Cosmos#2757] [Q&A]
Can someone make step by step please i dont know C# well i wish someone help me

**Comment by NikitaPos** [comment]
yea i redded all dicussion also and sry for bed eng

**Comment by NikitaPos** [comment]
@quajak can u help me please

**Comment by NikitaPos** [comment]
also i want make mouse as well

**Comment by zarlo** [comment]
you should learn the basics of C# first

**Reply by NikitaPos** [reply]
ty for tip lol btw well do you know how to make GUI

**Comment by NikitaPos** [comment]
i will try hard by myself lol

**Comment by 9xbt** [comment]
oh god lord lol
